recipes/getdns/all/conanfile.py

```diff
@@ -0,0 +1,170 @@
+            return True
+        else:
+            return self.options.stub_only
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def requirements(self):
+        self.requires("openssl/1.1.1h")
+        if self._with_libev:
+            self.requires("libev/4.33")
+        if self.options.with_libevent:
+            self.requires("libevent/2.1.12")
+        if self.options.with_libuv:
+            self.requires("libuv/1.38.1")
+        if self.options.with_libidn2:
+            self.requires("libidn2/2.3.0")
+        if self.options.tls == "gnutls":
+            self.requires("nettle/6.2")
+            # FIXME: missing gnutls recipe
+            raise ConanInvalidConfiguration("gnutls is not (yet) available on cci")
+        if not self._stub_only:
+            # FIXME: missing libunbound recipe
+            raise ConanInvalidConfiguration("libunbound is not (yet) available on cci")
+
+    def build_requirements(self):
+        self.build_requires("pkgconf/1.7.3")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("getdns-{}".format(self.version), self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["OPENSSL_USE_STATIC_LIBS"] = not self.options["openssl"].shared
+        self._cmake.definitions["ENABLE_SHARED"] = self.options.shared
+        self._cmake.definitions["ENABLE_STATIC"] = not self.options.shared
+        self._cmake.definitions["ENABLE_STUB_ONLY"] = self._stub_only
+        self._cmake.definitions["BUILD_LIBEV"] = self._with_libev
+        self._cmake.definitions["BUILD_LIBEVENT2"] = self.options.with_libevent
+        self._cmake.definitions["BUILD_LIBUV"] = self.options.with_libuv
+        self._cmake.definitions["USE_LIBIDN2"] = self.options.with_libidn2
+        self._cmake.definitions["USE_GNUTLS"] = self.options.tls == "gnutls"
+        self._cmake.definitions["BUILD_TESTING"] = False
+
+        self._cmake.configure()
+        return self._cmake
+
+    def build(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+        # Use FindOpenSSL.cmake to let check_function_exists succeed
+        # Remove other cmake modules as they use FindPkgConfig
+        for fn in glob.glob("Find*cmake"):
+            if "OpenSSL" not in fn:
+                os.unlink(fn)
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("COPYING", src=self._source_subfolder, dst="licenses")
```

Comment:
Should this be `LICENSE`, instead of the symlinked `COPYING`?

Reply:
Indeed, but I didn't pass `symlinks=True` to `self.copy`.
So not the symlink is copied, but the destination file (=`LICENSE`).
So in the end it's ok.